src/ducks/balance/Balance.jsx

```diff
@@ -253,18 +254,28 @@ class Balance extends PureComponent {
   handleResume() {
     this.updateQueries()
   }
-
+  onVisibleHandler() {
+    if (document.visibilityState === 'visible') {
+      this.handleResume()
+    }
+  }
   startResumeListeners() {
     if (__TARGET__ === 'mobile') {
       document.addEventListener('resume', this.handleResume)
       window.addEventListener('online', this.handleResume)
+    } else {
+      window.addEventListener('online', this.handleResume)
```

Comment:
nit: on peut donc sortir le `window.addEventListener('online', this.handleResume)` du if